Makefile

```diff
@@ -113,13 +113,19 @@ remove-with-olm: ## Remove controller from the K8s cluster
 
 .PHONY: bundle
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+	rm -rf ./bundle
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
-	cd config/manifests/bases && $(KUSTOMIZE) edit add annotation --force 'olm.skipRange':"$(SKIP_RANGE)" && \
+	cd config/default && \
+		$(KUSTOMIZE) edit set image kube-rbac-proxy=$(RBAC_PROXY_IMG) && \
+		$(KUSTOMIZE) edit set namespace $(OPERATOR_NAMESPACE) && \
```

Comment:
After removing this `set namespace` line from the Makefile i can deploy the operator in a different namespace. @jarrpa can you remove this line?

Reply:
Looks like something wrong with my cluster. everything works fine, sorry for the noise.

Reply:
Over a call, we discovered that something may have gone wrong in Madhu's environment. We verified on both our ends that, after making sure our shell environments were correct, the bundle installation worked correctly.